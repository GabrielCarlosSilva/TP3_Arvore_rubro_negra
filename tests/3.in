1 
2
Barbara 
24
Vitor
23
2
1
3
Karol
18
Bernardo
12
Marlon
28
0