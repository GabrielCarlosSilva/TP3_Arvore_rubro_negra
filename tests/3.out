Dados inOrder:
Nome: Vitor
Idade: 23
Nome: Barbara
Idade: 24
Dados inOrder:
Nome: Bernardo
Idade: 12
Nome: Karol
Idade: 18
Nome: Vitor
Idade: 23
Nome: Barbara
Idade: 24
Nome: Marlon
Idade: 28